### PR TITLE
Add Counter PDA Tutorial - Easy 

### PR DESCRIPTION
**Pull request by cleon30**
I have created a Counter PDA Tutorial  for the Solana Playground. This will be an easy tutorial to get a better understanding of accounts, PDAs, and  how basic operations work on Solana Anchor Programs.  

Is designed to be an introductory tutorial, where the user will have to fill the gaps in functions following the instructions from the pages of the tutorial.

### CODE REVIEW

File: client/src/tutorials/Template_counter/pages/2.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Structure of the Program 
+
+As you probably have seen, our initial program is divided in 3 main sections: Functions, Data Validators and Data Structures.
+
+- **Smart Contract Functions**: a section with the operations and logic of the program. 
+
+- **Data Validators**: a section for all the accounts that we will have to create and declare. As you can see, the *Context*<Name> from the functions is actually the same as the Data validators.
+
+- **Data Structures**: the structure of the accounts / rust structs that we will use to store info! In the program is pretty clear, the Counter account will store 2 things:
+```bash
```
Comment (acheroncrypto) [inline client/src/tutorials/Template_counter/pages/2.md:10]: `bash` should be `rust`

**Comment by acheroncrypto** [comment]
Thank you so much for making this! I've requested a small change and could you please run `yarn format` because CI will fail with the current formatting.

File: client/src/tutorials/Template_counter/pages/2.md (hunk 1 above)
Comment (acheroncrypto) [reply client/src/tutorials/Template_counter/pages/2.md:10]: Also ideally, the images live under the tutorial's folder, not the existing tutorial template folder. You can create `counter-easy` folder inside `public/tutorials` to keep things more organized.

**Review by acheroncrypto — APPROVED** [review]

**Comment by acheroncrypto** [comment]
Merged! Thanks again, one more small thing: it would be better to name the folder with React's naming convention `CounterEasy` instead of `Template_counter`. It's not a big deal but it's nice to have a standart naming convention.

**Comment by cleon30** [comment]
Perfect!! I will change the folder name and pull request again.

Thanks!!

**Pull request merged**